lib/msf/ui/console/driver.rb

```diff
@@ -601,6 +605,15 @@ def handle_ssh_ident(val)
     $VERBOSE = verbose
   end
 
+  def handle_session_tlv_logging(val)
+    val = val.downcase
+    if val && (val == 'console' || val == 'false' || val =~ /^file:.+/)
+      return true
+    end
```

Comment:
What are your thoughts on just defaulting `true` to mean `console`? In the future the semantics of `true` could be changed, i.e. console as well as to `~/.msf4/` 🤔